### ISSUE TITLE
Fedora 38: `GLIBC_2.35' not found

### ISSUE THREAD

**Issue by huubvanniekerk** [OpenShot/openshot-qt#5327]
**Describe the bug:**
After upgrading from Fedora 37 where it worked great, OpenShot-v3.1.1-daily-11583-95eccafc-08c2cdd1-x86_64.AppImage doesn't start anymore. In the terminal it shows:

OpenShot-v3.1.1-daily-11583-95eccafc-08c2cdd1-x86_64.AppImage 
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/cx_Freeze/initscripts/__startup__.py", line 40, in run
  File "/usr/local/lib/python3.8/dist-packages/cx_Freeze/initscripts/Console.py", line 37, in run
  File "openshot_qt/launch.py", line 55, in <module>
    from PyQt5.QtWidgets import QApplication
ImportError: /tmp/.mount_OpenShVuIhNw/usr/bin/libm.so.6: version `GLIBC_2.35' not found (required by /lib64/libharfbuzz.so.0)

**Steps to reproduce the behavior:**
start the appimage in the terminal to view the error....

**Expected behavior:**
OpenShot to open and function properly.

**System Details:**
- OpenShot-v3.1.1-daily-11583-95eccafc-08c2cdd1-x86_64.AppImage 
- Operating System / Distro: Fedora Linux 38

**Log Files:**
If you are experiencing a crash, please collect and attach logs of the problem.  Log files can be found in the `.openshot_qt` folder in your user home directory. Log files over 2 MB in size will need to be compressed, please attach a ZIP file instead of the raw logs.

**Exception / Stacktrace:**
If you are experiencing a crash or strange output on the command line, please attach a small snippet of the exception here, but please keep it as small as possible (for readability).

**Screenshots:** (Optional)
If applicable, add screenshots to help explain your problem. To include screenshots, copy/paste them here, drag the image(s) into this form, or click the link below this edit field to load images saved to your computer. All images are public, so please don't post screenshots containing personal information.

**Comment by huubvanniekerk** [comment]
Could the workaround from #5300 please be included in the next daily-build? In the meantime, I have installed version 3.1.1-4 from RPMFusion, which starts without a problem. It's behaviour I have to test yet.

Update 02-10-2023: using the rpmfusion version has a very annoying detail: each time when you import a file, you have to go from your home directory to the directory containing the file. The app-image stay in the chosen directory until you select another one.
So from that perspective, I really do prefer the AppImage.

**Comment by Colorjet3** [comment]
Hello @huubvanniekerk.  The lead developer has to look into including the workaround, or better yet a more permanent fix into the next daily build and eventually, the next production release.  I will assign this to him.

Regarding the importing issue, what version gets installed/running using RPMFusion (by the way, I know nothing about this)?  It could be that the AppImage is a newer version.  If you go to Edit | Preferences you will see a tab named Location.  In here you can set the default behavior of Importing Files, Saving a Project, and Exporting a Video.

**Comment by huubvanniekerk** [comment]
Hi @Colorjet3. Thank you for assigning it.

Regarding RPMFusion: I didn't know about it either until I read about it somewhere (this forum or somewhere?) The version stated is 3.1.1, with libopenshot 0.3.2.

Unfortunately, setting Importing Files didnt change the behaviour (yet), but maybe I'm still doing something wrong. 
Although with the AppImage, I never had to set this. It just worked the way I wanted and as such I don't know its (default) setting.

**Comment by Colorjet3** [comment]
Thank you @huubvanniekerk for the FYI regarding the RPMFusion.  We'll have to wait and hear back from the developer.

**Comment by huubvanniekerk** [comment]
@Colorjet3 I just tried OpenShot-v3.1.1-daily-11627-2034f45d-08c2cdd1-x86_64.AppImage from 9 October but unfortunately it ends with the same error: "ImportError: /tmp/.mount_OpenShlxRktd/usr/bin/libm.so.6: version `GLIBC_2.35' not found (required by /lib64/libharfbuzz.so.0)"

**Comment by RodBarnes** [comment]
Attempted to run it on Fedora 39 Cinnamon.  Same result there with the same error.  Is there an update on the status from the "lead developer"?

**Comment by Colorjet3** [comment]
Hello @jonoomph.  This needs your attention as we are seeing more and more of these in the past few months.

**Comment by elestopadov** [comment]
I have a problem too, with Ubuntu 23

**./OpenShot-v3.1.1-x86_64.AppImage** 

Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/cx_Freeze/initscripts/__startup__.py", line 40, in run
  File "/usr/local/lib/python3.8/dist-packages/cx_Freeze/initscripts/Console.py", line 37, in run
  File "openshot_qt/launch.py", line 55, in <module>
    from PyQt5.QtWidgets import QApplication
ImportError: /tmp/.mount_OpenShAwXCNa/usr/bin/libm.so.6: version `GLIBC_2.35' not found (required by /lib/x86_64-linux-gnu/libharfbuzz.so.0)

**Comment by huubvanniekerk** [comment]
I upgraded to Fedora 39 but before I could do that I had to uninstall packages. One of them was unfortunately OpenShot. Reinstallation comes with a dependency problem: 
"MariaDB-common-10.4.17-1.fc31.x86_64.rpm: Downloading successful, but checksum doesn't match."
It turns out that it's not only OpenShot depending on it. E.g. Pitivi does too. Solution anyone?
 
Edit: why is MariaDB a dependency on Fedora 39 but not on Fedora 38?
